dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/tasks/Ds3DeleteFilesTaskTest.java

```diff
@@ -45,31 +60,66 @@
     private final Workers workers = new Workers();
     private Session session;
     private boolean successFlag = false;
+    private final static ResourceBundle resourceBundle = ResourceBundle.getBundle("lang", new Locale(ConfigProperties.getInstance().getLanguage()));
+    private static final Ds3Client client = Ds3ClientBuilder.fromEnv().withHttps(false).build();
+    private static final Ds3ClientHelpers HELPERS = Ds3ClientHelpers.wrap(client);
+    private static final String TEST_ENV_NAME = "DeleteFilesTaskTest";
+    private static final String DELETE_FILES_TASK_TEST_BUCKET_NAME = "DeleteFilesTaskTest_Bucket";
+    private static final BuildInfoServiceImpl buildInfoService = new BuildInfoServiceImpl();
+    private static TempStorageIds envStorageIds;
+    private static UUID envDataPolicyId;
 
     @Before
     public void setUp() {
         new JFXPanel();
         Platform.runLater(() -> {
-            final SavedSession savedSession = new SavedSession(SessionConstants.SESSION_NAME,
-                    SessionConstants.SESSION_PATH, SessionConstants.PORT_NO, null,
-                    new SavedCredentials(SessionConstants.ACCESS_ID, SessionConstants.SECRET_KEY),
+            final SavedSession savedSession = new SavedSession(
+                    TEST_ENV_NAME,
+                    client.getConnectionDetails().getEndpoint(),
+                    "80",
+                    null,
+                    new SavedCredentials(
+                            client.getConnectionDetails().getCredentials().getClientId(),
+                            client.getConnectionDetails().getCredentials().getKey()),
+                    false,
                     false);
             session = new CreateConnectionTask().createConnection(
-                    SessionModelService.setSessionModel(savedSession, false));
+                    SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
+            try {
+                envDataPolicyId = IntegrationHelpers.setupDataPolicy(TEST_ENV_NAME, false, ChecksumType.Type.MD5, client);
+                envStorageIds = IntegrationHelpers.setup(TEST_ENV_NAME, envDataPolicyId, client);
+            } catch (IOException e) {
+                e.printStackTrace();
```

Comment:
If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.